Functions/test_function1.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from Functions.function1 import get_even_numbers
```

Comment:
De preferat, in Python pachetele, variabilele si functiile se numesc folosid litere mici si notarea snake_case